api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/spnego/SpnegoProperties.java

```diff
@@ -143,4 +143,14 @@ public class SpnegoProperties implements Serializable {
     @NestedConfigurationProperty
     private WebflowAutoConfigurationProperties webflow = new WebflowAutoConfigurationProperties().setOrder(100);
 
+    /**
```

Comment:
I think you may want to elaborate a little bit on the purpose of pool. What is it, what does it do and why do we now need it.

Reply:
Also, is this something that someone would ever want to configure? Do we need to expose this settings, or could we pick reasonable defaults and hard-code?

Reply:
Sure. I will update the Javadoc.

Reply:
I think we need both: provide default settings which make sense and make them configurable if need be.

For the timeout, I'm confident with 2 seconds. It should cover most of the cases.
For the size of the pool, I propose 10 as a default value, but maybe we could set 20 to give higher limits.

Reply:
OK, sure. Makes sense.